R/install-bitbucket.r

```diff
@@ -56,20 +80,10 @@ remote_download.bitbucket_remote <- function(x, quiet = FALSE) {
   if (!quiet) {
     message("Downloading bitbucket repo ", x$username, "/", x$repo, "@", x$ref)
   }
-
   dest <- tempfile(fileext = paste0(".zip"))
   src <- paste("https://bitbucket.org/", x$username, "/", tolower(x$repo), "/get/",
     x$ref, ".zip", sep = "")
-
-  if (!is.null(x$password)) {
-    auth <- httr::authenticate(
-      user = x$auth_user %||% x$username,
-      password = x$password,
-      type = "basic")
-  } else {
-    auth <- NULL
-  }
-
+  auth <- x$auth_token %||% NULL
```

Comment:
There is no point in having `%||% NULL`, the right hand side will only be used if the left hand side is already null, so you should just pass x$auth_token directly to download.